content/manuals/engine/storage/bind-mounts.md

```diff
@@ -268,7 +321,6 @@ Stop the container:
 
 ```console
 $ docker container stop devtest
-
 $ docker container rm devtest
```

Comment:
nit; if we want to simplify, this could be `docker container rm -f devtest` (or  `-fv`)